### ISSUE TITLE
Expected training time

### ISSUE THREAD

**Issue by Seth-Park** [YueLiao/CDN#20]
Hi,

Thank you so much for sharing this awesome work!
I was wondering how long it usually takes (i.e. hours or days) to train CDN given the settings explained in the paper.
Also, do you know how sensitive the model is to batch size? For instance, if I wanted to iterate faster and use a larger batch size (while changing the lr accordingly) for training, would that significantly affect the performance? 

I guess I could run these experiments myself and figure it out, but I wanted to know in advance before launching jobs given the already limited compute resources.

Thanks!

**Comment by YueLiao** [comment]
HI,

Thx for your interest in our work. The CDN-s is required to train 90 epochs for about 12 hours based a 16 batch sizes and 8 Tesla V100 GPUs on HICO-Det. Fewer epochs may cause some performance drop but not significant, where we have trained 60 epochs with decreasing the learning rate in 40 epochs, which only drops less than 0.5% mAP. A larger or a small batch size with a proper learning rate may not significantly affect the performance.
